src/App.js

```diff
@@ -25,45 +28,51 @@ class App extends React.Component{
     })
   }
   handleClose = () => {
-    this.setState ({
-      show:false
+    this.setState({
+      show: false
+    })
+  }
+  hornedFilter = (event) => {
+    event.preventDefault()
+    let newHorned = parseInt(event.target.value)
+    let newVar = this.state.data.filter(beast => beast.horns === newHorned)
+    this.setState({
+      filteredBeasts: newVar,
     })
+    console.log(newVar);
   }
-  
-  render(){
-    return(
+
+
+  render() {
+    return (
       <>
         <Header />
-        <SelectedBeast 
-          show={this.state.show}
-          handleClose={this.handleClose}
-          animal={this.state.animal}
-        />
+          <Container>
+            <Form onChange={this.hornedFilter}>
+              <Form.Group controlId="selected">
+                <Form.Select  >
+                  <option>Select How Many Horns You would like to See?</option>
+                  <option value="1">1 Horn</option>
+                  <option value="2">2 Horns</option>
+                  <option value="3">3 Horns</option>
+                  <option value="100">100 Horns</option>
+                </Form.Select>
+
+              </Form.Group>
+            </Form>
+          </Container>
```

Comment:
- We may want to consider if the Form should live here, live elsewhere , or if we can make it a separate component. 👍